### DISCUSSION TITLE
My GitHub request has been approved but I've not received my education benefits even after 72 Hours.

### DISCUSSION THREAD

**Discussion by Zaross** [community/community#138770] [GitHub Education]
### Hi everyone,

Hi everyone,

My verification has been complete and my request has been approved, although, I have not received my education benefit even after 72 hours after the approval.

**Comment by davevad93** [comment]
Hi @Zaross ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Zaross was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
